events/iq_board.py

```diff
@@ -0,0 +1,94 @@
+import discord
+from discord.ext import commands
+import utils.logging as log
+import utils.colours as colour
+import utils.values as value
+import utils.database as sql
+from utils.helpers import *
+
+
+class IQBoard(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+        self.iq_emoji = "<:WeSmart:682352802202386472>"
+        self.iq_requirement = 7
```

Comment:
I'm wondering if these are things we can include in values?

Reply:
I put it there for convenience sake if we ever need to change something, especially the requirement as I'm not sure how easy or hard it is to get 7 points.